### PR TITLE
Get current selection bounds on key navigation.

### PR DESCRIPTION
**Pull request by cthurston**
Fixes issue #184.

Changes proposed in this pull request:

- 
- 
-

### CODE REVIEW

**Comment by cthurston** [comment]
I needed access to the NativeEvent in beforebeginedit so that I could prevent default enter from occurring in the newly created editor.

**Comment by cthurston** [comment]
Noticed the controlInput was not being removed from the DOM.  Also, removed the copy/paste listeners.  I didn't test this on child grids to see if it would break anything.

File: lib/dom.js [hunk 1]
```diff
@@ -204,7 +204,7 @@ define([], function () {
             if (!(cell && cell.header)) { return; }
             //HACK for IE10, does not like literal enum
             enu = cell.header['enum'];
-            if (self.dispatchEvent('beforebeginedit', {cell: cell})) { return false; }
+            if (self.dispatchEvent('beforebeginedit', {cell: cell, NativeEvent})) { return false; }
```
Comment (TonyGermaneri) [inline lib/dom.js]: Can you change this to es5 syntax?  There's no transpiler hooked up to build and IE11 is a target.  Otherwise, looks great.

Comment (cthurston) [reply lib/dom.js]: Sure thing.

**Comment by TonyGermaneri** [comment]
THANKS! @cthurston Great catch and fix.  I'll get this tested and deployed today.

**Pull request merged**